DOCSP-34184: Atlas Search indexes (#99)
mongodb/docs-rust: source/whats-new.txt

The {+driver-short+} v2.8 release includes the following features,
improvements, and fixes:

> - Adds MongoDB Atlas Search index management methods. For more information about
>   these methods, see the :ref:`rust-atlas-search-indexes` guide.

Reviewer: ```suggestion
- Adds MongoDB Atlas Search index management methods. To learn more about
  these methods, see the :ref:`rust-atlas-search-indexes` guide.
```
